Seohee/Lv2/멀쩡한 사각형.js

```diff
@@ -0,0 +1,13 @@
+function solution(w, h) {
+    var answer = 0;
+    let dots = [];
+    function solution(w, h) {
+        let answer = 0;
+
+        for (let x = 1; x <= w; x++) {
+            const front_y = (-1) * (h) * (x - 1) / w + h;
+            const end_y = (-1) * (h) * x / w + h;
+            answer += (Math.ceil(front_y) - Math.floor(end_y));
```

Comment:
```suggestion
            const frontY = (-1) * (h) * (x - 1) / w + h;
            const endY = (-1) * (h) * x / w + h;
            answer += (Math.ceil(frontY) - Math.floor(endY));
```
Math.ceil이라는 좋은 메서드 배워갑니다!!!